public/chat_flyout.tsx

```diff
@@ -86,52 +87,55 @@ export const ChatFlyout = (props: ChatFlyoutProps) => {
     >
       <>
         <div className={cs('llm-chat-flyout-header')}>
-          <ChatWindowHeader />
+          {props.overrideComponent ? <ChatOverrideHeader /> : <ChatWindowHeader />}
```

Comment:
Is this `ChatOverrideHeader` suitable for all override component header? Shall we just hide the `ChatWindowHeader` and let `overrideComponent` take over the whole screen?

Reply:
Yes, I think this is a standard header for all override component. As UX required, we should change the title, add a back button and hide the history button. There is a mock up for AD, which also applies to alert.

![image](https://github.com/user-attachments/assets/8e3ac13b-cbab-4624-9a6b-8d629f5e60c8)